picard/ui/options/plugins.py

```diff
@@ -230,6 +232,20 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def plugin_remove(self):
+            plugin = self.items[self.ui.plugins.selectedItems()[0]]
+            buttonReply = QtWidgets.QMessageBox.question(self,'Confirmation', "Do you want to remove?",
```

Comment:
Do this feature actually uninstall a plugin (and remove related files from disk) ?
As i said in a previous comment in a previous PR, using "uninstall" instead of "remove" seems better to me (since we actually "install", the reverse operation is "uninstall").

Also the title of the dialog box should say "Uninstall plugin?" and message something like "Do you really want to uninstall the plugin pluginname ?".